internal/acceptance/acceptance.go

```diff
@@ -2,25 +2,43 @@ package acceptance
 
 import (
 	"fmt"
+	"log"
 	"os"
 	"testing"
 
+	"github.com/joho/godotenv"
+
 	"github.com/databrickslabs/databricks-terraform/common"
 	"github.com/databrickslabs/databricks-terraform/provider"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
 func AccTest(t *testing.T, tc resource.TestCase) {
+	loadFromDotEnvFile()
```

Comment:
why do we need to load env from file? that makes it impossible to run more than one integration test session in parallel. e.g. azure cli + awsst + mws

Reply:
removed